disnake/client.py

```diff
@@ -225,7 +225,7 @@ class Client:
         The total number of shards.
     application_id: :class:`int`
         The client's application ID.
-    intents: :class:`Intents`
+    intents: Optional[:class:`Intents`]
```

Comment:
nit: don't document this as optional here, it's planned to become required soon.

Reply:
~~Should that change not be part of the pr in which we make it required, then?~~

Reply:
~~*fine.*~~